From 91b1840731bc223fdfa0b86b0d9b1904213e3299 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Povi=C5=A1er?= <povik+lin@cutebit.org>
Date: Tue, 15 Mar 2022 19:41:05 +0100
Subject: [PATCH 083/171] ASoC: macaudio: State missing simple-card-utils
 dependency

State that the macaudio module depends on simple-card-utils to fix build
errors of the kind:

ld.lld: error: undefined symbol: asoc_simple_parse_routing
>>> referenced by macaudio.c:59 (sound/soc/apple/macaudio.c:59)
>>>               soc/apple/macaudio.o:(macaudio_snd_platform_probe) in archive
---
 sound/soc/apple/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/apple/Kconfig b/sound/soc/apple/Kconfig
index a3f6a97140e2..ce6188aa659d 100644
--- a/sound/soc/apple/Kconfig
+++ b/sound/soc/apple/Kconfig
@@ -3,6 +3,7 @@ config SND_SOC_APPLE_SILICON
 	tristate "ASoC machine driver for Apple Silicon Macs"
 	depends on ARCH_APPLE || COMPILE_TEST
 	select SND_SOC_APPLE_MCA
+	select SND_SIMPLE_CARD_UTILS
 	default ARCH_APPLE
 	help
 	  This option enables an ASoC machine driver for Apple Silicon Macs.
-- 
2.34.1

